MangoAPI.BusinessLogic/ApiCommands/Users/RegisterCommandHandler.cs

```diff
@@ -20,21 +21,40 @@ public class RegisterCommandHandler : IRequestHandler<RegisterCommand, Result<To
     private readonly IJwtGenerator jwtGenerator;
     private readonly IJwtGeneratorSettings jwtGeneratorSettings;
     private readonly IBlobServiceSettings blobServiceSettings;
+    private readonly PasswordHashService passwordHashService;
+    private readonly IMangoUserSettings mangoUserSettings; 
 
+    private readonly UserEntity mangoUser = new()
+    {
+        PhoneNumber = "56272381753",
+        DisplayName = "Mango Messenger",
+        DisplayNameColour = DisplayNameColour.Violet,
+        Bio = "Service notifications",
+        Id = SeedDataConstants.MangoId,
+        UserName = "MangoMessenger",
+        Email = "mango.messenger@wp.pl",
+        NormalizedEmail = "MANGO.MESSENGER@WP.PL",
+        EmailConfirmed = true,
+        PhoneNumberConfirmed = true,
+        Image = "mango_logo.png",
+    };
+    
     public RegisterCommandHandler(
         IUserManagerService userManager,
         MangoDbContext dbContext,
         IJwtGenerator jwtGenerator,
         IJwtGeneratorSettings jwtGeneratorSettings,
         ResponseFactory<TokensResponse> responseFactory,
-        IBlobServiceSettings blobServiceSettings)
+        IBlobServiceSettings blobServiceSettings, PasswordHashService passwordHashService, IMangoUserSettings mangoUserSettings)
```

Comment:
Following code style: move each of parameters `passwordHashService`, `mangoUserSettings` on new line